src/Tasks/Common/Resources/Strings.resx

```diff
@@ -948,5 +948,9 @@ You may need to build the project on another operating system or architecture, o
     <value>NETSDK1214: PreferNativeArm64 applies only to .NET Framework targets. It is not supported and has no effect for when targeting .NET Core.</value>
     <comment>{StrBegin="NETSDK1214: "}</comment>
   </data>
-  <!-- The latest message added is Net8NotCompatibleWithDev177. Please update this value with each PR to catch parallel PRs both adding a new message -->
+  <data name="TargetFrameworkIsNotRecommended" xml:space="preserve">
+    <value>NETSDK1215: Targeting .NET Standard prior to 2.0 is no longer recommended. See {0} for more details.</value>
+    <comment>{StrBegin="NETSDK1215: "}</comment>
```

Comment:
I assume `NETSDK1215` is new ID specific for targeting .NET Standard < 2.0? If so, I like it. If not, I think we should have a specific diagnostic.

Also, I assume the URL is derived from this ID?

Reply:
The aka.ms link is hardcoded in the targets themselves and included in the {0} location in the message: https://aka.ms/dotnet/dotnet-standard-guidance <-- let me know if you prefer this or a different link and then we should get the url created.

The NETSDK1215 is just an incrementing number for the SDK message that allows customers to search for it and suppress it. We try to update our documentation each major release with the list of warnings we have and include separate documentation for any that warrant it: https://learn.microsoft.com/en-us/dotnet/core/tools/sdk-errors/

Reply:
No that works. As long as the ID is unique for this diagnostic that's fine; the URL doesn't need to have it -- I just didn't see the URL so I assumed it's a generic scheme.